site/content/specs.md

```diff
@@ -33,24 +33,24 @@ as well as how to classify a request for a feature we don't yet support.
 
 ### WebAssembly Core
 wazero conforms with tests defined alongside WebAssembly Core
-Specification [1.0][1] and [2.0][14]. This is the default [RuntimeConfig][7].
+Specification [1.0][1] and [2.0][14].
 
-The WebAssembly Core Specification [2.0][2] is in draft form and wazero has
-[work in progress][8] towards that. Opt in via the below configuration:
+By default, the runtime configuration only enables WebAssembly 1.0 features, but
+you can opt in via the below configuration:
 ```go
 rConfig = wazero.NewRuntimeConfig().WithWasmCore2()
 ```
 
-One current limitation of wazero is that it doesn't fully implement the Text
-Format, yet, e.g. compiling `.wat` files. The intent is to [finish this][9],
-and meanwhile users can work around this using tools such as `wat2wasm` to
+One current limitation of wazero is that it doesn't implement the Text
+Format, e.g. compiling `.wat` files. Users can work around this using tools such as `wat2wasm` to
 compile the text format into the binary format. In practice, the text format is
 too low level for most users, so delays here have limited impact.
 
-#### Post 2.0 Features
+#### Post 1.0 Features
```

Comment:
Maybe leave as post 2.0 because it happens to be the case that all finished proposals are in the 2.0 draft. there may be years again between 2.0 and 3.0, and the same issue remains.. something not assigned to a release.

Ex we'll be asked about >2.0 (Ex rust already uses phase 2 threads proposal)

```suggestion
#### Post 2.0 Features
```